lib/firefox/webdriver/builder.js

```diff
@@ -199,6 +199,11 @@ module.exports.configureBuilder = function (builder, baseDir, options) {
     }
   }
 
+  if (options.debug) {
```

Comment:
Think we need to check for not running on Android? Or do the devtools switch work on Android?

Reply:
Also I can add the same for Chrome.